From f915fad33c702704c55bfc285534fea67e237c22 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Thu, 12 Jan 2012 04:34:50 +0100
Subject: [PATCH 205/217] units: make sure syslog socket goes away early
 during shutdown (cherry picked from commit
 ead51eb4ed55981f290e40a871ffbca6480c4cd3)

---
 units/syslog.socket |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/units/syslog.socket b/units/syslog.socket
index 323fa86..657e791 100644
--- a/units/syslog.socket
+++ b/units/syslog.socket
@@ -11,6 +11,8 @@
 Description=Syslog Socket
 DefaultDependencies=no
 Before=sockets.target syslog.target
+Conflicts=shutdown.target
+Before=shutdown.target
 
 # Pull in syslog.target to tell people that /dev/log is now accessible
 Wants=syslog.target
-- 
1.7.8.3

